src/wp-includes/html-api/class-wp-html-active-formatting-elements.php

```diff
@@ -149,7 +148,7 @@ public function remove_node( $token ) {
 	 *     > EM -> STRONG -> A ->
 	 *
 	 * To start with the most-recently added element and walk towards the top,
-	 * @see WP_HTML_Active_Formatting_Elements::walk_up
+	 * {@see WP_HTML_Active_Formatting_Elements::walk_up}.
```

Comment:
```suggestion
	 * see WP_HTML_Active_Formatting_Elements::walk_up().
```

For functions methods, the developer.wordpress.org docs parser will pick up links if brackets are included, `{@see` is only needed for hooks. See `get_posts()` for an example:

https://github.com/WordPress/wordpress-develop/blob/444bdf49bc38883f603a584bcc119aedf98257b8/src/wp-includes/post.php#L2392

:1234: Applies throughout.

Reply:
Are you suggesting we remove the brackets? If so, I have a couple questions:
 - although the docs parser might pick them up, what constitutes a link and how can we tell before the docs are published and updated if we did it right?
 - IDE support catches the `{@see …}` syntax as do all normal docblock parsing code. This is lost when removing the brackets. Are you saying you prefer removing the brackets and losing that support?
 - Are there any particular benefits to removing the brackets and the standard docblock tagging?

Reply:
> * lthough the docs parser might pick them up, what constitutes a link and how can we tell before the docs are published and updated if we did it right?

* For linking to hooks `{@see 'some_hook_name'}` constitutes a link (the quotes are required)
* For linking to functions and methods `function()` and `Class::method()` constitute links (the brackets are required)

You can validate that you've done it correctly but running [the docs parser](https://github.com/WordPress/phpdoc-parser) locally. Rather than generating the full docs, it's best to specify the file when running the command to avoid waiting for the full docs to be generated `wp parser create /path/to/file.php --user=<id|login>`

> Are you suggesting we remove the brackets?

Yeah, I'm suggesting removing the braces, and adding the brackets to the method names -- reviewing the WP Code it seems to be the standard. I'm basing this on a ticket opened by a meta team member after I was similarly unclear, see [WP#45445](https://core.trac.wordpress.org/ticket/45445). The key quote being:

> The documentation standards [mention](https://make.wordpress.org/core/handbook/best-practices/inline-documentation-standards/php/#other-descriptions) the use of @see for referencing hooks, but could stand to be more explicit about not needing to be used for functions, classes, or class methods.

Reply:
> running [the docs parser](https://github.com/WordPress/phpdoc-parser) locally

thank you! I've asked before how I can preview the docs and this is the first time someone has indicated how. I'll poke around with that. seems like it needs to run on a copy of WordPress as a plugin.

> I'm suggesting removing the braces, and adding the brackets to the method names 

looks like I was confused on the distinction of braces and brackets and parenthesis.

I'm really bummed about removing syntax that works and breaks IDE integrations for that hides the links implicitly. do you know there's a preference not to use the standardized docblock syntax?

if I'm required to remove the curly brackets I'm more inclined to awkwardly rewrite the docblock so it can use a `@see` tag on its own. I'd be sad if people are prevented from navigating the code because this specific formalization isn't required in order for the docs to display correct.

any thoughts on this? is there wiggle-room to cater towards developers who are reading the code?

Reply:
I thought there was nothing in the inline documentation standards that indicates "the right way" to do this, but I did see something in [this section](https://developer.wordpress.org/coding-standards/inline-documentation-standards/php/#phpdoc-tags):

> @see | elementname | References another function/method/class the function/method relies on. **Should only be used inline for “linking” hooks.**
(my emphasis)

There's certainly a difference between "should" and "must", and Core has instances of inline `@see` referencing functions. I'm not sure there's a solid answer to this one yet. We could probably do with some clarification in the standards.

Reply:
> I'm really bummed about removing syntax that works and breaks IDE integrations for that hides the links implicitly. do you know there's a preference not to use the standardized docblock syntax?

I'm really basing this on none of the existing code in core using `@see` for functions/methods (some external libraries do) and the ticket I linked to above.

FWIW, I think your point about supporting IDEs is a solid argument for starting a discussion on make/core. Apparently I don't have the right extension installed in VSCode for `@see` linking to work but it makes sense that IDEs would use it.

> I'll poke around with that. seems like it needs to run on a copy of WordPress as a plugin.

I recall it being a little fiddly to get set up but worth it for testing potential edge cases.

Reply:
Are we good to go with the syntax as written and continue considering the special syntax for WordPress in follow-up work that focuses on that PHPDoc tag and whether to allow the inline form?

Reply:
I'd rather go with the existing standard in WP core, ie `see WP_HTML_Active_Formatting_Elements::walk_up()` and the equivalent elsewhere. A make/core post can be used to discuss whether we change the standard.

If this PR uses a special case, all that is going to happen is someone will do a follow up commit fixing the coding standards for consistency.

Reply:
I will update to the existing pattern, though I thought we established it's not truly a standard; more like nothing else in Core has used this legitimate pattern.

Will try and post this quickly so it doesn't hold up the patch. Thanks @peterwilsoncc

Reply:
updated in cdd785cbbf

the language is awkward with `see ` coming out of nowhere, so I had to smudge a few places and use a normal `@see` tag to avoid making it confusing, and we've lost the ability to lean on tools provided by the IDE, but the inline syntax is gone.